### PR TITLE
Refactorings discovered during stewards etl

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: src/main/java/gov/acwi/wqp/etl/summaries/monitoringLocationSum/CreateMonitoringLocationSumPkTasklet.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.batch.core.StepContribution;
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.core.scope.context.ChunkContext;
+import org.springframework.batch.core.step.tasklet.Tasklet;
+import org.springframework.batch.repeat.RepeatStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+//TODO
```
Comment (mbucknell) [inline src/main/java/gov/acwi/wqp/etl/summaries/monitoringLocationSum/CreateMonitoringLocationSumPkTasklet.java]: Is there other work left todo here? Is there a ticket for this work?

File: src/main/resources/db/changelog/functions/transform/monitoringLocationSum/transformMonitoringLocationSum.sql [hunk 2]
```diff
@@ -59,13 +59,13 @@ begin
...
                                                station_id,
-                                               characteristic_type,
+                                               coalesce(characteristic_type, ''Not Assigned'') characteristic_type,
                                                sum(case when years_window = 1 then total_results else null end) past_12_months,
                                                sum(case when years_window < 6 then total_results else null end) past_60_months,
                                                sum(total_results) all_months
                                           from ml_grouping_swap_stewards
                                          where grouping_id = 1001
-                                            group by data_source_id, station_id, characteristic_type
+                                            group by data_source_id, station_id, coalesce(characteristic_type, ''Not Assigned'')
```
Comment (mbucknell) [inline src/main/resources/db/changelog/functions/transform/monitoringLocationSum/transformMonitoringLocationSum.sql]: I don't understand why this is necessary?

File: src/main/java/gov/acwi/wqp/etl/summaries/monitoringLocationSum/CreateMonitoringLocationSumPkTasklet.java (hunk 1 above)
Comment (dsteinich) [reply src/main/java/gov/acwi/wqp/etl/summaries/monitoringLocationSum/CreateMonitoringLocationSumPkTasklet.java]: WQP-1395

File: src/main/resources/db/changelog/functions/transform/monitoringLocationSum/transformMonitoringLocationSum.sql (hunk 2 above)
Comment (dsteinich) [reply src/main/resources/db/changelog/functions/transform/monitoringLocationSum/transformMonitoringLocationSum.sql]: The grouping doesn't like null values, so we are defaulting to 'Not Assigned' for the type. you also must group by the values in the query...

**Pull request merged**